docs/Documentation/Applications/chemicalKinetics.md

```diff
@@ -0,0 +1,284 @@
+
+|                                                                          | C++   | Fortran | Python | Matlab | GPU    | Speed*$^5$ | Features | Cost | Compatibility       | Speciality                   |
+|:------------------------------------------------------------------------:|:-----:|:-------:|:------:|:------:|:------:|:----------:|:--------:|:----:|:-------------------:|:----------------------------:|
+| [Cantera](https://cantera.org/)                                          | y     | y       | y      | y      | x      | ++         | ++++     | Free | Research codes*$^8$ | Simplicity, large user base  |
+| [zero-RK](https://github.com/LLNL/zero-rk)                               | y     | x       | x*$^6$ | x      | y*$^1$ | ++++*$^7$	 | ++*$^4$  | Free | Converge CFD ($)    | Model reduction tools        |  
+| [PelePhysics](https://amrex-combustion.github.io/PelePhysics/)           | y     | x       | x      | x      | y      | +++++      | +++      | Free | Amrex/Pele          | HPC, NREL popular framework  |
+| [Chemkin Pro](https://www.ansys.com/products/fluids/ansys-chemkin-pro)   | y     | y*$^2$  | x      | x      | x*$^3$ | ++++       | ++++     | $    | Ansys ($)           | Legacy, professional support |
+
+
+## Strategy
+A typical workflow could look as follows:
+
+1. Create mechanisms and validate with Cantera:
+
+	👍 Feature rich, multi language, very [well documented](https://cantera.org/documentation/index.html), large [support forum](https://groups.google.com/g/cantera-users).
+
+	👎 Slower than competition, no GPU support.
+
+
+2. Perform model reduction with zero-RK if necessary:
+
+	👍  Faster than Cantera & Chemkin for [more than 100 species](https://ipo.llnl.gov/sites/default/files/2019-09/zork.pdf), some [GPU support](https://doi.org/10.1115/ICEF2017-3631).
+
+	👎 Fewer features, sparse documentation, C++ only.
+
+3. Convert to a high performance C++ code with PelePhysics and link to a CFD solver: 
+
+	👍  [GPU](https://amrex-combustion.github.io/PelePhysics/CvodeInPP.html#cvode-implementation-in-pelephysics-on-gpu), [well documented](https://amrex-combustion.github.io/PelePhysics/index.html), [amrex/Pele CFD](https://amrex-combustion.github.io/) & Cantera mechanisms compatible.
+	
+	👎 Fewer features than Cantera & Chemkin, C++ only. 
+
+## Installation and testing on Kestrel
+
+### Cantera
+!!! note
+	Cantera can be [installed from source](https://cantera.org/install/compiling-install.html#sec-compiling) or with the conda envionment on the Kestrel as explained below.
+	The performance can vary depending on the compile options and flags while compiling from source. We are more than happy to learn from power users about the flags which lead to the best performance. 
```

Comment:
Could add an instruction about who users should contact with recommendations. Would you like them to contact you directly via email, or hpc-help@nrel.gov?